apps/condo/domains/organization/integrations/sbbol/sync/syncBankAccounts.js

```diff
@@ -0,0 +1,80 @@
+const { getLogger } = require('@open-condo/keystone/logging')
+const get = require('lodash/get')
+
+const { initSbbolFintechApi } = require('@condo/domains/organization/integrations/sbbol/SbbolFintechApi')
+const { dvSenderFields } = require('@condo/domains/organization/integrations/sbbol/constants')
+const { BankAccount } = require('@condo/domains/banking/utils/serverSchema')
+const { RUSSIA_COUNTRY } = require('@condo/domains/common/constants/countries')
+const { ISO_CODES_FOR_SBBOL } = require('@condo/domains/common/constants/currencies')
+const { getSchemaCtx } = require('@open-condo/keystone/schema')
+const { Organization } = require('@condo/domains/organization/utils/serverSchema')
+
+const logger = getLogger('sbbol/syncBankAccounts')
+
+const _syncBankAccount = async (ClientInfo) => {
+    const { keystone: userContext } = await getSchemaCtx('User')
+    const { keystone: bankAccountContext } = await getSchemaCtx('BankAccount')
+    const [organization] = await Organization.getAll(userContext,
```

Comment:
are you trying here to get oldest organization? if you don't nedd others, please add limit